docs/frequently-asked-questions.md

```diff
@@ -16,68 +16,68 @@
+* If you are familiar with Vagrant, or willing to learn how to setup Vagrant + Homestead and you have a machine capable of running a virtual machine and vagrant, then go with [the vagrant based setup](https://docs.ushahidi.com/platform-developer-documentation/getting-started/setup_alternatives/vagrant-setup) \(This is the setup Ushahidi staff uses on a daily basis!\)
+* If you are a frontend developer that wants to contribute without setting up the backend, then try [setting up the platform client](https://docs.ushahidi.com/platform-developer-documentation/getting-started/setup_alternatives/setting-up-the-platform-client) only, and using the API route of an ushahidi.io deployment.  This the fastest way to get started.
 
-Are you ready to deploy Ushahidi for others to use? 
+__Are you ready to deploy Ushahidi for others to use?__
 
-* Do you plan to modify the code in some way? Then follow the [Installing for production environments setup guide.](https://docs.ushahidi.com/platform-developer-documentation/getting-started/setup_alternatives/installing-for-production-environments)
-* Do you want to deploy Ushahidi and not worry about upgrading, monitoring services, or hosting it yourself ? Then start a new deployment in [http://ushahidi.io/create](http://ushahidi.io/create) and let us manage all the tech for you.
-* Something in between? [Contact the Ushahidi team ](http://gitter.im/ushahidi/Community)and we'll figure it out! We're always happy to chat.
+* if you plan to modify the code in some way, follow the [Installing for production environments setup guide.](https://docs.ushahidi.com/platform-developer-documentation/getting-started/setup_alternatives/installing-for-production-environments)
+* If you want to deploy Ushahidi without worrying about upgrading, monitoring services, or hosting it yourself, start a new deployment in [http://ushahidi.io/create](http://ushahidi.io/create) and we will manage all the tech for you.
+* If you're looking for something in between, [Contact the Ushahidi team ](http://gitter.im/ushahidi/Community)and we'll figure it out! We're always happy to chat.
 
 ### How do I install Ushahidi?
 
-Please follow the step by step guide on how to install Ushahidi [here](https://www.ushahidi.com/support/install-ushahidi).
+A step by step guide on how to install Ushahidi can be found [here](https://www.ushahidi.com/support/install-ushahidi).
 
 ### Can I install the Platform API?
 
-Yes you can. You can follow this link for more resources on how to install [here](https://ushahidi.gitbook.io/platform-developer-documentation/~/edit/drafts/-LU01pd_YH9N3j-BhwFw/tech-stack-overview/api-documentation).
+Yes you can. You can follow this link for more resources on how to install: [here](https://ushahidi.gitbook.io/platform-developer-documentation/~/edit/drafts/-LU01pd_YH9N3j-BhwFw/tech-stack-overview/api-documentation).
```

Comment:
oops our link there is broken, if you can fix it, that'd be great! 
https://docs.ushahidi.com/platform-developer-documentation/tech-stack/api-documentation 

otherwise I will do it after merging :) no worries